.circleci/config.yml

```diff
@@ -627,6 +627,7 @@ jobs:
         environment:
           - SERVICES=mssql
           - PLUGINS=tedious
+          - DD_TRACE_DISABLED_PLUGINS=mocha
```

Comment:
This should not be needed. If it breaks for us it means it will break for others too.

Reply:
as spoken in Slack, I was testing whether `node-tedious` was testing because of the changes, but the errors are in other branches as well